runners/flink/src/test/java/org/apache/beam/runners/flink/FlinkPipelineExecutionEnvironmentTest.java

```diff
@@ -418,4 +426,20 @@ private FlinkPipelineOptions setPipelineOptions(
             })
         .collect(Collectors.toList());
   }
+
+  private List<URL> getJars(Object env) throws Exception {
+    try {
+      return (List<URL>) Whitebox.getInternalState(env, "jarFiles");
+    } catch (Throwable t) {
```

Comment:
We shouldn't be catching throwable here.

```suggestion
    } catch (FieldNotFoundException e) {
```